src/components/ReportActionItem/MoneyRequestPreview.tsx

```diff
@@ -232,19 +223,16 @@ function MoneyRequestPreview(props) {
             return translate('iou.receiptScanning');
         }
 
-        if (TransactionUtils.hasMissingSmartscanFields(props.transaction)) {
+        if (TransactionUtils.hasMissingSmartscanFields(transaction)) {
             return Localize.translateLocal('iou.receiptMissingDetails');
         }
 
         return CurrencyUtils.convertToDisplayString(requestAmount, requestCurrency);
     };
 
-    const getDisplayDeleteAmountText = () => {
-        const {amount, currency} = ReportUtils.getTransactionDetails(props.action.originalMessage);
-
-        if (isDistanceRequest) {
-            return CurrencyUtils.convertToDisplayString(TransactionUtils.getAmount(props.action.originalMessage), currency);
-        }
+    const getDisplayDeleteAmountText = (): string => {
+        const iouOriginalMessage: IOUMessage | EmptyObject = action?.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? action.originalMessage : {};
+        const {amount = 0, currency = CONST.CURRENCY.USD} = iouOriginalMessage;
 
         return CurrencyUtils.convertToDisplayString(amount, currency);
     };
```

Comment:
During this file migration, it turned out that `originalMessage` of action is passed to methods where `transaction` is expected to get the amount and currency values. Though `originalMessage` doesn't look like transaction at all
![image](https://github.com/Expensify/App/assets/23176449/d2a087b1-af22-480e-a120-f31cfb426b34)
After some testing I've find out that the functionality of `getDisplayDeleteAmountText` function can be simplified.